### DISCUSSION TITLE
Elementary OS 7 Screenshots not copying to clipboard

### DISCUSSION THREAD

**Discussion by Oohapinkcar** [elementary/screenshot-tool#267] [Q&A] [answered]
Ive read several topics about this but unsure about whether or not this is a bug or if an update is needed. I need to shift+print and have the screenshot copied to the clipboard not as a file in screenshots. 

Elementary OS 7 
Linux 5.19.0-45-generic

**Comment by jeremypw** [comment]
I just tried this with the daily version of Elementary OS 7 and it worked OK - I could paste the screenshot from the clipboard into a GitHub comment for example.  I guess the image may not be in the required format for some apps? Which app are you trying to paste into?

**Reply by Oohapinkcar** [reply]
@jeremypw Ive tried pasting into Gmail and Slack. I also just tried to paste in Github and same thing.

**Reply by Oohapinkcar** [reply]
@jeremypw I took a screenshot using the app and it did let me select copy to clipboard. It just doenst allow me to do it via keyboard shortcut. I guess thats a work around.

**Comment by jeremypw** [comment]
Looks like this commit implemented screenshot to clipboard capability: https://github.com/elementary/gala/commit/491b3cc6e5938a46ece4eb73766deea3cfdc7e20

The required shortcut is `<Control><PrtSc>`  not shift + print.

**Reply by Oohapinkcar** [reply]
Yea the <control+prtscrn> allows me to copy from clipboard but its a full print screen not a selected area. At least i found a work around but using the app directly. Thanks for the help @jeremypw

**Reply by Oohapinkcar** [reply, marked as answer]
@jeremypw I just saw it in the shortcuts. Copy an area to clipboard <Shift+Ctrl+Prntscrn>. Thank you sir

**Comment by jeremypw** [comment]
`<Shidt><Control><PrtSc>` allows you to select an area to be copied to the clipboard.  `<Control><Alt><PrtSc>` allows you to copy the current window to the clipboard.  These shortcuts (and others) can be seen by pressing and releasing the `<Window>` key.  AH I see you discovered it while I was writing!

**Reply by Oohapinkcar** [reply]
Thanks for the help bud
